api/soap/mantisconnect.wsdl

```diff
@@ -747,6 +814,30 @@
   <part name="per_page" type="xsd:integer" /></message>
 <message name="mc_filter_get_issue_headersResponse">
   <part name="return" type="tns:IssueHeaderDataArray" /></message>
+<message name="mc_filter_search_issue_headersRequest">
+  <part name="username" type="xsd:string" />
+  <part name="password" type="xsd:string" />
+  <part name="filter" type="tns:FilterSearchData" />
+  <part name="page_number" type="xsd:integer" />
```

Comment:
In first version of the new searching method it was as a part of a filter. We discussed it with @vboctor and finally we did it the same way as the other filtering method (mc_filter_get_issues etc.).